package.json

```diff
@@ -84,6 +84,9 @@
     ]
   },
   "prettier": "@backstage/cli/config/prettier",
+  "jest": {
+    "rejectFrontendNetworkRequests": true
```

Comment:
What's the reason for this change? Seems unrelated 🤔

Reply:
Just me being bad with git. Fixed.